app/src/ui/branches/pull-request-list.tsx

```diff
@@ -82,28 +67,34 @@ export class PullRequestList extends React.Component<
   }
 
   public componentWillReceiveProps(nextProps: IPullRequestListProps) {
-    if (nextProps.pullRequests === this.props.pullRequests) {
-      return
```

Comment:
I can trace this change back to the initial PR list work #2823, but given we need to handle other prop changes I don't think we should be exiting early at all...